tests/functional/runner.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Process\Process;
+
+require_once 'vendor/autoload.php';
+
+$finder = new Finder();
+$finder->in(__DIR__)->directories()->depth(0);
+
+$log = function ($message): void {
```

Comment:
jakej je tady rozdíl oproti `function log...`?

Reply:
globální vs lokální funkce. function log se mi zaregistruje kdoví kam do globálního namespacu. Když mi bude někde ten log překážet tak prostě unset($log) a je to. 

Normálně by to byl nějaký logger v SF console app. A teoreticky by to šlo nahradit za 

```
class Logger {
 public function __invoke($message) {$this->log($message)}
};
$log = new Logger();
```

Ale nemá to nějaký zásadní důvod, jen nemám rád globální funkce

Reply:
Namespace to taky vyřeší. Ale je to fuk, jen mě to zajímalo.